stories/core/TimePicker.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { action } from '@storybook/addon-actions';
+import { useEffect, useState } from '@storybook/addons';
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+import {
+  TimePicker,
+  TimePickerProps,
+  IconButton,
+  LabeledInput,
+  MeridiemType,
+} from '../../src/core';
+import SvgCalendar from '@itwin/itwinui-icons-react/cjs/icons/Calendar';
+
+export default {
+  title: 'Core/TimePicker',
+  component: TimePicker,
+  argTypes: {
```

Comment:
Could use `date: { control: { type: 'date' } }` for adding date picker control 😄

Reply:
Added